subscriptions/package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "@binaris/shift-fetch-runtime": "0.0.3",
-    "@binaris/shift-interfaces": "0.1.2",
+    "@binaris/shift-interfaces": "0.2.0-rc.0",
```

Comment:
We should probably not reference this package directly anymore.

Reply:
As above: we actually take from there.